next-tavla/app/(admin)/components/IllustratedInfo/index.tsx

```diff
@@ -14,7 +14,7 @@ function IllustratedInfo({
         <div className={classes.info}>
             <Image src={animals} aria-hidden="true" alt="" />
             <Heading2 className={classes.infoHeading}>{title}</Heading2>
-            <LeadParagraph>{description} </LeadParagraph>
+            <LeadParagraph className="pb-4 mt-0">{description} </LeadParagraph>
```

Comment:
Er det kanskje bedre å sette en padding rundt hele `<div> `som wrapper alt dette? Slipper man å være avhengig av at teksten er til stede for at boksen skal ha en padding